### ISSUE TITLE
Game Mechanic - Wisp Companion 

### ISSUE THREAD

**Issue by NimNimR4r** [TeaylThorn/HighwynnGame#4]
**Describe the Bug Part A**
Players found it difficult to read the advice given by the wisp companion, it would appear next to objects that you wouldn't associate with speech (like the dragon fruit), which made them a bit confused as to what was going on in the level 

***
**Version**
Version One 

***
**To Reproduce**
Players would:
1. Walk/run up to certain objects (in this case the dragon fruit/pink box) 
2. Spot the text when near an object and would either try and read it and follow the directions, or be slightly confused... 
3. It would work for them and they'd understand where the text was coming from, or still be confused... 

***
**Expected Behaviour**
Players were expecting the text to be more orientated to the wisp companion, instead of the objects in scene, some thought the fruit was talking or some of the enemies >> 

***
**Additional Content**
A way to fix this and make it more clear, or to make it more noticeable that the wisp is talking to the dragon (player) would be to have a text box come up at the bottom of the screen with the text running across it (Issun from Okami could be a good starting point for ideas) 

***
***
**Describe the Bug Part Two**
Less of a bug and more of an update, players would like to be able to use the wisp as a weapon and attack enemies through the companion as well (the boomerang effect). 
Once the wisp was described to our play testers they were really intrigued and wanted to see more of what the wisp could do 
(see the [Player & Character Mechanics](https://github.com/TeaylThorn/HighwynnGame/wiki/Player-&-Character-Mechanics) page for info/keys)

***
**TLDR**
* Update Wisp's speech to be more noticeable to player, instead of showing up and floating around the scene/objects (Issun from Okami could be good for ideas) 
* Update Wisp's mechanic so that players can use it (e.g. boomerang, posses, light the way)
